staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_restart.go

```diff
@@ -63,6 +63,9 @@ var (
 	        Resource rollout will be restarted.`))
 
 	restartExample = templates.Examples(`
+		# Restart all deployments in default namespace
+		kubectl rollout restart deployment
```

Comment:
nit: would be better to add an example by explicitly specifying a namespace such as `kubectl rollout restart deployment -n test-namespace`